src/components/Logo/Logo.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
```

Comment:
Just a tiny request: could we rename this to "RecogitoLogo" for a more explicit name? BTW: I note that there is a "RecogitoLogo" component currently. But that's an outdated placeholder we can (finally) drop.